orchagent/Makefile.am

```diff
@@ -62,6 +62,8 @@ orchagent_SOURCES = \
             chassisorch.cpp \
             debugcounterorch.cpp \
             natorch.cpp \
+            mlagorch.cpp \
```

Comment:
@Praveen-Brcm , @gitsabari,
In https://github.com/Azure/sonic-swss/pull/1349/files we use `mclag` instead of `mlag`. Would be good to follow the same naming convention across the feature.